### DISCUSSION TITLE
Update billing info: Displays Test Mode, hides current card

### DISCUSSION THREAD

**Discussion by frogandcode** [plausible/analytics#689] [Anything Else]
I navigated to _Settings | Update billing_ info to check which credit card I'm using and potentially update it. Upon doing so, I see this:

![image](https://user-images.githubusercontent.com/12514/107118617-037b0b80-6850-11eb-86a8-ca81fe384990.png)

A couple issues:

- The fact that it says "Test mode" / "View testing instructions" makes me nervous that something is wrong and perhaps I'm in a sandbox environment
- I'm unable to see the current credit card that I'm using

The second issue is more of an annoyance (I can update the card even if it's already in use), but I'm leery to change my card and upgrade my account without knowing that this form is pointing to the right Paddle environment.

**Comment by frogandcode** [comment]
UPDATE: Apparently the "Test mode" issue was a global problem at the payment provider (Paddle), not with Plausible. Sorry about the false alarm there.

I'd still like to be able to see my current billing info, but if Paddle doesn't support this and it's not a high priority to make it happen some other way, I understand–feel free to close this issue.

**Comment by metmarkosaric** [comment]
thanks for the update @frogandcode! i'll place this in feature requests now.
